packages/web/src/components/organisms/AuthForm/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react'
+import { Row, Col, Form, Input, Button } from 'antd'
+import { useMarketScheme, useAuthenticate } from 'src/fixtures/dev-kit/hooks'
+import { useEffectAsync } from 'src/fixtures/utility'
+
+export const AuthForm = ({ property }: { property: string }) => {
+  const [schemeList, setSchemeList] = useState<string[]>([])
+  const { marketScheme } = useMarketScheme()
+  const { authenticate } = useAuthenticate()
+  const onFinish = (values: any) => authenticate(property, Object.values(values))
+
+  useEffectAsync(async () => {
+    const schemeList = await marketScheme()
+    setSchemeList(schemeList || ['test-placeholder']) // for development
+  }, [])
```

Comment:
I couldn't get the market scheme in my environment, so I defined the default placeholder for development. I would love for you to review the implementation of useMarketScheme, etc. And I hope you find the cause of the bug:bow: